src/style.css

```
@@ -0,0 +1,64 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :nth-child(-n+4) {
+  margin-right: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
```

let's move `background-image` here
```suggestion
  background: url(images/star.svg) no-repeat center;
```